deploy/provision.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd -P -- "$(dirname -- "$0")/terraform" || exit
+
+echo "WARNING:"
+echo "This will create a new EC2 instance. Do not use this script if you are trying to deploy new code."
```

Comment:
I think it would be nice to add an option like  "Press x to exit" or something so ppl don't just have to ctrl-c to exit the script if they run it accidentally

Reply:
`terraform apply` already allows you to type `yes` or `no` to continue execution so I am actually going to remove get rid of one of these `read -rp` items - it's not hard to exit the flow of this script. I have `terraform apply || exit` because I'm assuming that answering no will stop the script (I will be appending configuration items to this script - it doesn't actually matter rn because it is the last item anyway) .